frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -598,6 +598,33 @@ class Worksheet extends React.Component {
             return;
         }
         const item = this.refs.list.refs['item' + index];
+
+        // Make the judgement only if the shouldScroll parameter has the default value of `True`
+        // Logic: Only scroll when the target element is completely not on the screen
```

Comment:
Can you mention a bit more in the comment about what purpose this serves? (i.e., making sure that the screen doesn't scroll when you normally press j / k, until the target element is completely not on the screen)

Reply:
Have made the modification!